### DISCUSSION TITLE
Import same module in differents js or ts files

### DISCUSSION THREAD

**Discussion by luc-git** [wailsapp/wails#2934] [Q&A]
Hello I have just a question is it possible to import runtime.js module in differents ts files because I'm trying to do this and I have the error: "TypeError: can't access property "EventsOnMultiple", window.runtime is undefined".

**Comment by leaanthony** [comment]
https://wails.io/docs/guides/frontend/#overriding-default-script-injection

**Reply by luc-git** [reply]
It doesn't works. I included runtime.js in two ts files

**Reply by leaanthony** [reply]
It has to work as that's how the initial page gets it. Did you check that your app is actually requesting the files in the network tab?

**Reply by luc-git** [reply]
WDYM by network tab?

**Reply by luc-git** [reply]
![Screenshot 2023-09-23 002704](https://github.com/wailsapp/wails/assets/102831178/2bad0dab-4346-4111-a5ae-5ccfa579334e)

Ok so index.html + main.ts actually requests both runtime.js and ipc.js

![Screenshot 2023-09-23 002734](https://github.com/wailsapp/wails/assets/102831178/d954d618-c73b-4a0f-b65c-8c85970d8bd4)

explorer.html + explorer.ts (second html and ts files) only requests runtime.js and not ipc.js

Maybe the problem is here

**Reply by luc-git** [reply]
If I correctly understood network tab is the tab in "inspect element" in web browsers

**Reply by luc-git** [reply]
Ok I fixed the initial error by adding 

`<script src="/wails/runtime.js"></script>`
`<script src="/wails/ipc.js"></script>`

in explorer.html (the second html file).
But now js console in inspect element gives me this error:

`Uncaught TypeError: window.WailsInvoke is not a function`
